src/hooks/useLoyaltyPass.ts

```diff
@@ -41,52 +35,28 @@ export const useLoyaltyPass = (): UseLoyaltyPassProps => {
       account?.address?.toLowerCase() !== storedAddress?.toLowerCase());
 
   // query
-  const apiBaseUrl = process.env.NEXT_PUBLIC_GATEWAY_URL;
-  const apiUrl = new URL(`${apiBaseUrl}`);
-  const apiKey = process.env.NEXT_PUBLIC_GATEWAY_API_KEY;
-  const apiAccesToken = process.env.NEXT_PUBLIC_GATEWAY_API_TOKEN;
-  const headers = {
-    'x-api-key': `${apiKey}`,
-    Authorization: `Bearer ${apiAccesToken}`,
-  };
-
+  const apiBaseUrl = process.env.NEXT_PUBLIC_JUMPER_API;
   const { data, isSuccess, isLoading } = useQuery({
     queryKey: ['loyalty-pass'],
     queryFn: async () => {
-      const res = await request(
-        decodeURIComponent(apiUrl.href),
-        getAllPDAs,
-        { EVMAddress: account?.address },
-        headers,
-      );
+      const res = await fetch(`${apiBaseUrl}/users/${account?.address}`);
 
-      if (res && account?.address) {
-        let points = 0;
-        let tier = '';
-        const { issuedPDAs: pdas } = res as IGatewayAPI;
-        // filter to remove loyalty pass from pda
-        const pdasWithoutLoyalty = pdas.filter((pda: PDA) => {
-          if (pda.dataAsset.title === 'LI.FI Loyalty Pass') {
-            points = pda.dataAsset.claim.points;
-            tier = pda.dataAsset.claim.tier;
-            return false;
-          }
-          return true;
-        });
+      const data = await res.json();
```

Comment:
let's add a pre-check to make sure `res` can be jsoned, something like this should be sufficient:

```
  if (!res.ok) {
    // handle error
 } 

  const data = await res.json();
  ```